src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -187,7 +187,8 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     )
 
     val message = executionResult match {
-      case SuccessfulExecution(outputs, returnCode, hash) => WorkflowActor.CallCompleted(key, outputs, returnCode, hash)
+      case SuccessfulExecution(outputs, returnCode, hash) =>
+        WorkflowActor.CallCompleted(key, outputs, returnCode, if (workflowDescriptor.writeToCache) hash else "")
```

Comment:
I'm finding myself agreeing with @geoffjentry that we should not stringly type the hash, `""` is not clear as an empty hash